infra/feast-operator/config/component_metadata.yaml

```diff
@@ -0,0 +1,4 @@
+releases:
```

Comment:
should we add a comment at the top of this file explaining why its necessary?

Reply:
done